test/run_test

```diff
@@ -34,6 +35,8 @@ test -n "${OS-}" || false 'make sure $OS is defined'
 CIDFILE_DIR=$(mktemp --suffix=postgresql_test_cidfiles -d)
 
 volumes_to_clean=
+s2i_args="--pull-policy=never "
```

Comment:
Used only once, shouldn't be used everywhere?  Could this be moved to `run_s2i_test`?